src/pages/HomePage/ListOfHouses/ListOfHouses.tsx

```diff
@@ -1,24 +1,21 @@
-/* eslint-disable import/no-absolute-path */
-/* eslint-disable import/no-unresolved */
-/* eslint-disable react/prop-types */
 /* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable @typescript-eslint/no-redeclare */
 import DoneIcon from '@mui/icons-material/Done';
 import FavoriteIcon from '@mui/icons-material/Favorite';
 import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
 import { Autocomplete, Box, Button, Checkbox, TextField } from '@mui/material';
 import Chip from '@mui/material/Chip';
 import { useEffect, useState } from 'react';
-
-import noPhoto from '/src/assets/images/nophoto.png';
+import noPhoto from 'src/assets/images/nophoto.png';
 
 import styles from './ListOfHouses.module.scss';
 
-const options = ['Payment (Low to High)', 'Payment (High to Low)', 'A-Z', 'Z-A'];
-const label = { inputProps: { 'aria-label': 'Checkbox demo' } };
+const options = ['Payment (Low to High)', 'Payment (High to Low)', 'A-Z', 'Z-A'] as const;
+const label: { inputProps: { 'aria-label': string } } = { inputProps: { 'aria-label': 'Checkbox demo' } };
 
-function ListOfHouses({ houses }) {
-  const [sortType, setSortType] = useState(null);
-  const [sortedHouses, setSortedHouses] = useState([]);
+function ListOfHouses({ houses }: { houses: BasicHouseType[] }) {
```

Comment:
Is your VSCode recognising the type `BasicHouseType`? My seems to not know where it came from :thinking:.